### PR TITLE
Adds cmd to remove projects without publisher from the ES index

### PR DESCRIPTION
**Pull request by paoloromolini**
Adds a command that removes from the index all projects without publisher projects.
It also prints the absolute url of the project on the screen so that one can debug it.

We could add other clean functions here in the future.

### CODE REVIEW

**Comment by paoloromolini** [comment]
After creating this I was wondering if we do want to index projects which don't have a publisher in the first place.

**Comment by francescozaia** [comment]
@paoloromolini nope. They shouldn't exist at all once we're live.

File: readthedocs/docsitalia/management/commands/clean_es_index.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+from readthedocs.projects.models import Project
+
+
+class Command(BaseCommand):
+
+    """Removes projects without publisher from the ES index"""
+
+    def handle(self, *args, **options):
+        """handle command"""
+        e_s = Elasticsearch()
```
Comment (xrmx) [inline readthedocs/docsitalia/management/commands/clean_es_index.py]: mi sa che devi passargli settings.ES_HOSTS

File: readthedocs/docsitalia/management/commands/clean_es_index.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        for p_o in Project.objects.filter(publisherproject__isnull=True):
```
Comment (xrmx) [inline readthedocs/docsitalia/management/commands/clean_es_index.py]: conviene filtrare anche per poublisher project e publisher non attivi

**Review by xrmx — COMMENTED** [review]
Alla fine forse conviene anche togliere dal db tutti i progetti che non hanno una fk verso un publisher_project

**Pull request closed without merging**